### DISCUSSION TITLE
Initial sync speed up

### DISCUSSION THREAD

**Discussion by screw** [OffchainLabs/prysm#11468] [Q&A] [answered]
Is it possible to speed up the initial sync (apart from using snapshot nodes)?
A few days ago we started up `prysm` on ETH mainnet and it was running at around `50 blocksPerSecond`. Since then the rate is dropping down and now it is running at around only`9 blocksPerSecond` and the ETR is 76hours.

- Is it _normal_/expected that the initial sync slows down so much?
- Can anybody suggest tips on parameters to fully utilise our machine?

HW: 48core CPU, 150GB RAM, NVMe storage, ...

**Comment by prestonvanloon** [comment, marked as answer]
It is normal for this type of speed regression. Phase 0 of the beacon chain was much easier to process blocks. As the number of validators and requirements of new phases of beacon chain increases, the sync speed will become slower.

See https://github.com/prysmaticlabs/prysm/issues/11476 for some additional context. 

One suggestion is to use checkpoint sync to start syncing from a trusted checkpoint.

https://docs.prylabs.network/docs/prysm-usage/checkpoint-sync

**Reply by screw** [reply]
Thank you. I've switched to checkpoint sync and it has been lightning fast.
